Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -389,3 +390,16 @@ extension PBXProductType {
         }
     }
 }
+
+extension Scheme.Test {
+    var systemAttachmentLifetime: XCScheme.TestAction.AttachmentLifetime? {
```

Comment:
Seeing as though these 2 booleans end up as an enum, we could also just add a simple `attachmentLifetime` enum  to the spec instead. Thoughts?

Reply:
My thought was that the "attachment lifetime" is more of an implementation detail of the XCScheme rather than the Scheme in the project spec. From the user's perspective, the bools are a 1:1 mapping with these two checkboxes in the Xcode scheme editor. Happy to discuss this more, though.
<img width="460" alt="Screen Shot 2020-09-06 at 12 07 11 PM" src="https://user-images.githubusercontent.com/1727803/92330034-e9879380-f039-11ea-853c-9439c860b7f4.png">

Reply:
@yonaskolb any more thoughts on this? I'm happy to make this change if you think it's the right call.